### PR TITLE
Reordered and renamed ScreenNav items

### PR DESCRIPTION
**Pull request by samchan0221**
Ref #1392

### CODE REVIEW

File: portal/src/ScreenNav.tsx [hunk 1]
```diff
@@ -87,18 +90,8 @@ const ScreenNav: React.FC = function ScreenNav() {
...
-            textKey: "ScreenNav.password-policy",
-            url: "configuration/passwords/policy",
-          },
-          {
-            textKey: "ScreenNav.forgot-password",
-            url: "configuration/passwords/forgot-password",
-          },
-        ],
+        textKey: "ScreenNav.password-policy",
+        url: "configuration/passwords/policy",
```
Comment (louischan-oursky) [inline portal/src/ScreenNav.tsx:94]: Change the URL to `configuration/password-policy`

File: portal/src/ScreenNav.tsx [hunk 2]
```diff
@@ -116,18 +109,22 @@ const ScreenNav: React.FC = function ScreenNav() {
         textKey: "ScreenNav.localization",
         url: "configuration/localization",
       },
+      {
+        textKey: "ScreenNav.subscription",
+        url: "configuration/settings/subscription",
+      },
```
Comment (louischan-oursky) [inline portal/src/ScreenNav.tsx:115]: Rename the key to `ScreenNav.billing`.
Update the URL to `billing`

File: portal/src/ScreenNav.tsx [hunk 3]
```diff
@@ -116,18 +109,22 @@ const ScreenNav: React.FC = function ScreenNav() {
...
       {
         textKey: "ScreenNav.settings",
```
Comment (louischan-oursky) [inline portal/src/ScreenNav.tsx:117]: Update the key to `ScreenNav.Advanced`

File: portal/src/ScreenNav.tsx [hunk 4]
```diff
@@ -116,18 +109,22 @@ const ScreenNav: React.FC = function ScreenNav() {
...
         url: "configuration/settings",
```
Comment (louischan-oursky) [inline portal/src/ScreenNav.tsx:118]: Update the URL to `advanced`

File: portal/src/ScreenNav.tsx [hunk 5]
```diff
@@ -116,18 +109,22 @@ const ScreenNav: React.FC = function ScreenNav() {
...
         children: [
-          {
-            textKey: "PortalAdminSettings.title",
-            url: "configuration/settings/portal-admins",
-          },
           {
             textKey: "ScreenNav.sessions",
             url: "configuration/settings/sessions",
```
Comment (louischan-oursky) [inline portal/src/ScreenNav.tsx:122]: Update the URL to `advanced/sessions`

File: portal/src/ScreenNav.tsx [hunk 6]
```diff
@@ -116,18 +109,22 @@ const ScreenNav: React.FC = function ScreenNav() {
...
           },
+          {
+            textKey: "ScreenNav.forgot-password",
+            url: "configuration/passwords/forgot-password",
```
Comment (louischan-oursky) [inline portal/src/ScreenNav.tsx:126]: Update the URL to `advanced/forgot-password`

File: portal/src/ScreenNav.tsx [hunk 7]
```diff
@@ -116,18 +109,22 @@ const ScreenNav: React.FC = function ScreenNav() {
...
+          },
           {
             textKey: "ScreenNav.webhooks",
             url: "configuration/settings/web-hooks",
```
Comment (louischan-oursky) [inline portal/src/ScreenNav.tsx:130]: Update the URL to `advanced/webhooks`

File: portal/src/ScreenNav.tsx [hunk 8]
```diff
@@ -136,21 +133,14 @@ const ScreenNav: React.FC = function ScreenNav() {
             textKey: "ScreenNav.admin-api",
             url: "configuration/settings/admin-api",
```
Comment (louischan-oursky) [inline portal/src/ScreenNav.tsx:134]: Update the URL to `advanced/admin-api`

File: portal/src/ScreenNav.tsx [hunk 9]
```diff
@@ -136,21 +133,14 @@ const ScreenNav: React.FC = function ScreenNav() {
...
           },
-          {
-            textKey: "ScreenNav.subscription",
-            url: "configuration/settings/subscription",
-          },
         ],
       },
+      {
+        textKey: "PortalAdminSettings.title",
+        url: "configuration/settings/portal-admins",
```
Comment (louischan-oursky) [inline portal/src/ScreenNav.tsx:140]: Update the URL to `portal-admins`

**Review by louischan-oursky — COMMENTED** [review]
Update `configuration/dns/custom-domains` to `custom-domains`.

**MAKE SURE NO LINKS ARE BROKEN**
**MAKE SURE NO LINKS ARE BROKEN**
**MAKE SURE NO LINKS ARE BROKEN**

**Comment by louischan-oursky** [comment]
@samchan0221 I added two commits. You may have a look.

**Comment by samchan0221** [comment]
> @samchan0221 I added two commits. You may have a look.

That was my mistake

**Pull request merged**